onnxruntime/core/providers/cpu/nn/conv_transpose_attributes.h

```diff
@@ -181,11 +181,11 @@ struct ConvTransposeAttributes : public ConvAttributes {
       ORT_ENFORCE(*out_size >= 0);
       // total padding size
       int64_t paddings = std::max<int64_t>(0, (in_size - 1) * stride + adj + (kernel - 1) * dilation + 1 - *out_size);
-      if (pad_type == AutoPadType::SAME_UPPER) {  // pad more on head when paddings are odd.
+      if (pad_type == AutoPadType::SAME_LOWER) {  // pad more on head when paddings are odd.
```

Comment:
Could you please share some context on this toggle ? - As far as i can see, the way it currently is in master seems all right as per the spec - https://github.com/onnx/onnx/blob/master/docs/Operators.md#ConvTranspose. 

I think this change is causing tests to break.

Reply:
Here is more context: https://github.com/onnx/onnx/pull/3019
Actually the document in ONNX needs to be updated as well. I think that's the reason why ORT here has incorrect behavior in the first place...
I remember there is one deprecated model test failed. Let me rerun CIs. Thank you.